cli/drivers/ValetDriver.php

```diff
@@ -178,4 +178,35 @@ protected function isActualFile($path)
     {
         return ! is_dir($path) && file_exists($path);
     }
+
+    /**
+     * Load server environment variables if available.
+     * Processes any '*' entries first, and then adds site-specific entries
+     *
+     * @param  string  $sitePath
+     * @param  string  $siteName
+     * @return void
+     */
+    public function loadServerEnvironmentVariables($sitePath, $siteName)
```

Comment:
Is this method even used in our code-base?

Reply:
@jmarcher at this point no.  It should be used in the server.php to load any custom env variables.  But that file will take a bit of refactoring.  It's not hurting anything, I would say leave it in.